phpunit.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit
+    colors="true"
+    bootstrap="./vendor/autoload.php" type="coverage-text" target="tests/" charset="UTF-8">
+    
+    <testsuites>
```

Comment:
Are the target and type properties relevant here ? Please keep the line break format foreach properties